### PR TITLE
Default rebroadcast mode for Router and Repeater to ignore problematic portnums

### PR DESCRIPTION
**Pull request by thebentern**

### CODE REVIEW

File: src/mesh/Router.cpp [hunk 1]
```diff
@@ -591,19 +591,22 @@ void Router::handleReceived(meshtastic_MeshPacket *p, RxSource src)
...
+#endif
+        if (shouldIgnoreNonstandardPorts && p->which_payload_variant == meshtastic_MeshPacket_decoded_tag &&
             (p->decoded.portnum == meshtastic_PortNum_ATAK_FORWARDER || p->decoded.portnum == meshtastic_PortNum_ATAK_PLUGIN ||
              p->decoded.portnum == meshtastic_PortNum_PAXCOUNTER_APP || p->decoded.portnum == meshtastic_PortNum_IP_TUNNEL_APP ||
              p->decoded.portnum == meshtastic_PortNum_AUDIO_APP || p->decoded.portnum == meshtastic_PortNum_PRIVATE_APP ||
              p->decoded.portnum == meshtastic_PortNum_DETECTION_SENSOR_APP ||
              p->decoded.portnum == meshtastic_PortNum_RANGE_TEST_APP ||
              p->decoded.portnum == meshtastic_PortNum_REMOTE_HARDWARE_APP)) {
-            LOG_DEBUG("Ignoring packet on blacklisted portnum during event");
+            LOG_DEBUG("Ignoring packet on blacklisted portnum for CORE_PORTNUMS_ONLY");
```
Comment (caveman99) [inline src/mesh/Router.cpp:606]: don't we have this nifty is_one_of macro?

**Review by caveman99 — APPROVED** [review]

**Comment by caveman99** [comment]
Just for clarification, these are defaults, not runtime enforced? cause if i wanna have a router that will also forward audio, i would set the role and then switch the core packet restraint off again...

**Comment by thebentern** [comment]
> Just for clarification, these are defaults, not runtime enforced? cause if i wanna have a router that will also forward audio, i would set the role and then switch the core packet restraint off again...

Right. You set the Role to Router, then you go change the Rebroadcast mode to All after that.

File: src/mesh/Router.cpp (hunk 1 above)
Comment (thebentern) [reply src/mesh/Router.cpp:606]: Hah! I didn't even think about, but's a perfect application for it. Changes will be pushed shortly

**Pull request merged**